manifests/splunk-kubernetes-logging/configMap.yaml

```diff
@@ -14,12 +14,42 @@ data:
     @include source.journald.conf
     @include monit.conf
     @include output.conf
+    @include prometheus.conf
   system.conf: |-
     # system wide configurations
     <system>
       log_level info
       root_dir /tmp/fluentd
     </system>
+  prometheus.conf: |-
```

Comment:
I dont think its a good idea to enable this by default on the manifests. I assume people dont want it by default.

Reply:
It shouldn't take that much extra resources to expose this data. I'd actually be in favor of leaving it on (a lot of helm charts do). But! If you'd like them removed from the manifest, I can. Just me know.

Reply:
I think currently we can remove it from the manifests and update it if the community wants it.